sixtyfps_compiler/passes/visible.rs

```diff
@@ -24,9 +24,10 @@ pub fn handle_visible(component: &Rc<Component>, type_register: &TypeRegister) {
         component,
         &(),
         &mut |elem: &ElementRc, _| {
-            if elem.borrow().native_class().map_or(false, |n| {
+            let has_native_class = elem.borrow().native_class().map_or(false, |n| {
```

Comment:
that's not a right name for it.

Maybe `is_visibility_clip_element`  or `is_lowered_from_visible_property` ?